app/javascript/components/project_members/ProjectMembersTable.vue

```diff
@@ -133,7 +133,7 @@ export default {
     },
     available_members: {
       type: Array,
-      required: true,
+      required: false,
```

Comment:
Will non-admins be able to view the project members? If so, then when presenting the table to authors & reviewers we shouldn't have to provide available members to the component

Reply:
This was causing an error on the console since we are currently using the project members table in some locations without passing `available_members` but it is currently required.